Gordon360/Services/WellnessService.cs

```diff
@@ -0,0 +1,127 @@
+         
+            if (result == null)
+            {
+                 throw new ResourceNotFoundException() { ExceptionMessage = "The data was not found." };
+            }
+
+
+            var wellnessModel = result.Select(x =>
+            {
+                WellnessViewModel y = new WellnessViewModel();
+
+                DateTime currentTime = DateTime.Now;
+                
+                DateTime changeTime = new DateTime(currentTime.Year, currentTime.Month, currentTime.Day, 5, 0, 0);
+
+                // Answer is good until 5am the next day, so changeTime is most recent 5am
+                // which might be yesterday.
+                if(changeTime > currentTime)
+                {
+                    TimeSpan day = new TimeSpan(24, 0, 0);
+                    changeTime = new DateTime(changeTime.Ticks - day.Ticks);
+                }
+
+                if (x.timestamp >= changeTime)
+                {
+                    y.answerValid = true;
+                    y.userAnswer = x.userAnswer;
+                    y.timestamp = x.timestamp;
+                    return y;
+                }
+
+                y.answerValid = false;
+                y.userAnswer = x.userAnswer;
+                y.timestamp = x.timestamp;
+                
+                return y;
+            });
+
+
+
+            return wellnessModel;
+
+        }
+
+        /// <summary>
+        ///  Gets answer to the wellness check answer and sends it to the back end.
+        ///     If answer boolean is true: student is feeling symptomatic
+        ///     If answer boolean is false: student is not feeling symptomatic
+        /// </summary>
+        /// <param name="id">id</param>
+        /// <param name="answer">answer</param>
+        /// <returns>Ok if message was recorded</returns>
+
+        public WellnessViewModel PostStatus(bool answer, string id)
+        {
+            var _unitOfWork = new UnitOfWork();
+            var query = _unitOfWork.AccountRepository.FirstOrDefault(x => x.gordon_id == id);
+            if (query == null)
+            {
+                 throw new ResourceNotFoundException() { ExceptionMessage = "The account was not found." };
+            }
+
+            var idParam = new SqlParameter("@ID_NUM", id);
+            var answerParam = new SqlParameter("@Answer", answer);
+
+            var result = RawSqlQuery<WellnessViewModel>.query("INSERT_HEALTH_CHECK @ID_NUM, @Answer", idParam, answerParam); //run stored procedure
+            if (result == null)
+            {
+                 throw new ResourceNotFoundException() { ExceptionMessage = "The data was not found." };
+            }
+
+            var UserAnswer = answer;
+
+            WellnessViewModel y = new WellnessViewModel()
+            {
+                userAnswer = UserAnswer
+            };
+
+
+
+            return y;
+
+        }
+
+
+
+    }
+}
```

Comment:
Last line is missing an end of line.  It's ok to leave it, but better to add a newline.

Reply:
I'm confused as to what you mean by an end of line, I check some other service files and couldn't find anything that different than what I had. I supposed you meant just adding a blank line. If that is the case, than it's done. If you had another thing in mind, let me know haha.